### ISSUE TITLE
is bolt getting dumber?

### ISSUE THREAD

**Issue by TechIntegrationLabs** [stackblitz/bolt.new#4483]
### Describe the bug

i just gave bolt a prompt that asked to simply make the navbar transparent and the black text white, and it failed miserably.  This is a task that I am certain bolt could have handled perfectly a few weeks ago.  What is going on?   

I'm about to spend another $50 on bolt but not if it's not capable of doing simple things like this.

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-8tqdb4bu

### Validations

- [X] Please make your project public or accessible by URL. This will allow anyone trying to help you to easily reproduce the issue and provide assistance.

### Steps to reproduce

prompt it with this: make the navbar transparent and all black text white

### Expected behavior

the changes as described in the prompt

### Platform

```
Browser name = Chrome
Full version = 131.0.0.0
Major version = 131
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 67675861,
  "usedJSHeapSize": 64951453,
  "jsHeapSizeLimit": 4294705152
}
Username = TechIntegrationLabs
Chat ID = 82826938106d
```

**Comment by WarithNiallah** [comment]
Yes. I think they have reprogrammed it to waste tokens.  I have done some SIMPLE tests and it waits just until deployment and ends up in endless loops.
